lib/led/rgb.js

```diff
@@ -0,0 +1,189 @@
+module.exports = function(Led, priv) {
```

Comment:
Looking at the uses of `priv` in this file, I realized that you don't need to share the `priv` that's created in `Led`, so this file can just initialize it's own: `var priv = new Map();`